system-test/storage.js

```diff
@@ -1228,13 +1228,6 @@ describe('storage', function() {
           })
         );
 
-        it(
```

Comment:
I removed this because the URL can still be created, even without providing a userProject on a requester pays bucket. There was either an API change that now allows this, or something else was busted on our end.

Reply:
When the URL is created, are you still able to upload?

Reply:
Yes. We test this through `file.save()` flow, which performs a resumable upload.

Reply:
Sorry, I reversed the case we were talking about. The resumable upload does fail when a userProject isn't provided. I added a more thorough test to cover it now.

Reply:
So this is no longer an issue?

Reply:
thanks for clarifying. So, It isn't an issue until you attempt to the upload.

Reply:
Right-- after you get the URI, everything works as you would expect (fails without a userProject, and succeeds with).

Reply:
Thanks, waiting on more clarity on the case here. This is mildly confusing, but sounds as if it's WAI.

Reply:
I'm not sure if you meant clarity and confusion from my explanation, so just in case:

Our resumable uploads work in two phases: 1) Create a resumable URI, 2) Send data to it. When we first implemented requester pays support, step 1 would fail if the bucket was a requester pays bucket and no `userProject` was sent in the URL parameters. Somewhere along the way, our system tests started to fail, because step 1 began succeeding under the same conditions.

Ultimately, the way it works now seems correct-- a user can create a resumable URI without deciding what project will be billed for it, but is required to decide by the time data is actually uploaded. Nonetheless, since it's different than the way it was, I want to make sure :)